modules/vulnerability-management20-view-cve.adoc

```diff
@@ -45,12 +45,22 @@ a|
 a|
 * *Name*: The name of the CVE.
 * *Discovered time*: The date when {product-title-short} discovered the CVE.
-* *CVSS*: The severity level for the CVE. You can select from the following options for the severity level:
+* *CVSS*: The severity level for the CVE.
++
+The following values are associated with the severity level for the CVE:
++
 ** *is greater than*
 ** *is greater than or equal to*
 ** *is equal to*
 ** *is less than or equal to*
 ** *is less than*
+* *NVD CVSS*: The National Vulnerability Database (NVD) provides the severity level for the CVE.
++
+The following values are associated with the NVD severity level for the CVE: 
++
+** ??
```

Comment:
It should be 
_NVD CVSS*: The National Vulnerability Database (NVD) provides a CVSS score between 0 to 10 for a CVE based on the severity of the vulnerability_
you can add table on this link https://nvd.nist.gov/vuln-metrics/cvss